libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BeginDialog.schema

```diff
@@ -19,12 +19,19 @@
             ]
         },
         "dialog": {
-            "oneOf": [
+            "anyOf": [
                 {
                     "$kind": "Microsoft.IDialog",
                     "pattern": "^(?!(=)).*",
                     "title": "Dialog"
                 },
+                {
+                    "$ref": "schema:#/definitions/stringExpression",
+                    "examples": [
+                        "=concat('name', 'dialog')",
+                        "stringName"
+                    ]
+                },
                 {
```

Comment:
no, remove this, it was correct before, IDialog already says it can be a string.

Reply:
removed